ogr/services/github/service.py

```diff
@@ -99,7 +99,8 @@ def __init__(
                 max_retries=self._max_retries,
             )
 
-        logger.warning(f"Ignored keyword arguments: {kwargs}")
+        if kwargs:
+            logger.debug(f"Ignored keyword arguments: {kwargs}")
```

Comment:
I am thinking if it is really not worth a `warning`? Since it's printed only for non-empty `kwargs`, I would go with `warning`.

Reply:
I definitely saw examples where kwargs contained actual data: I recall username and max_retries.

Reply:
And that should be fixed on the called side I think. @mfocko WDYT?

Reply:
I'll change it back to warning: you're right we'll be able to correct the usage then